CharacterCreation/01_01_Races_Human

```diff
@@ -0,0 +1,145 @@
+==============================
+Humans
+==============================
+
+===== Society =====
+
+Humans originated on the planet Earth in the Sol system. They have been exploring space 
+to some degree for almost 800 years, settling their own Terran Empire after the catastrophic 
+events that changed the Earth.
+In general, adult humans range from 5’ to 6’5”. They can be larger or smaller, but individuals
+of those types are generally rare. 
+They are an incredibly prolific and opportunistic species, having completely terraformed and 
+colonized over half a dozen planets and their respective moons. With many more Terran planets 
+on the rise, they already dwarf the small Opalite and Tundari civilizations.
+
+Humans are very culturally dissimilar to each other. Even more so than most other species, 
+most likely due to their rapid expansion. As such, it is impossible to pick out the cultural 
+traits that influence humans as a whole. Central government is important to the large groups of 
+humans, though smaller factions and settlements on the rim of their empire do not seem to care 
+about the rules and regulations of their governing powers.
+
+
+Types:
+* Biological
+* Mammalian
+* Warm-blooded 
+* Omnivorous 
+
+
+===== Language: Various ===== 
+
+Most humans speak what is known as Terran Common. In addition, humans have countless different
+languages that they use to communicate with others of their peers. The majority of humans 
+speak multiple languages, whether alien or domestic. Humans are very good at repeating 
+sounds that are initially unnatural to them. The language centers of their brains are highly 
+developed and well-used. 
+Terran Common, Eastern Common, Westlich, and Mandar are the most common human languages.
+
+[Humans gain 1 additional language and pick up new languages at twice the normal rate]
+
+
+===== Names ===== 
+
+Humans can be named in as many ways as there are Terran languages. Usually, these names 
+have some significant meaning to the parents, but that is not always the case. There is no 
+standard of naming that is generally agreed upon.
+
+== Human Name Examples ==
+
+* Matthew Kesler
+* Lupita Ugara
+* Tomoyo Usagi
+
+===== Available Traits =====
+
+Pitching Arm:
+All humans have the inherent bone and muscle structure necessary for extremely 
+accurate throwing. Not all humans practice this technique, but you are one who 
+has mastered this seemingly simple aspect of your anatomy.
+[Negates the aiming action necessary to throw accurately]
```

Comment:
@bleehu When we've been playing CX, do we actually aim when throwing weapons?
I think I've had it both ways.

Reply:
I don't think people have been aiming with grenades especially. I didn't think we had a rule on it?

Reply:
In my experience, I don't know if it's written in the rules, but I'm pretty sure we've had a 'roll to hit what you're aiming at'. I could change it to reflect that?